test/regressions/fixtures/Masonry/EmptyMasonry.js

```diff
@@ -0,0 +1,6 @@
+import * as React from 'react';
```

Comment:
I think the unit test is better in this case. You can do something like
```js
it('should not throw if children are empty', () => {
  expect(() => render(<Masonry columns={3} spacing={1} />)).not.to.throw()
})
```

Reply:
Always love your code review, Jun.